themes/DeepWave.clinkprompt

```diff
@@ -0,0 +1,499 @@
+	["magenta_nb"] = {
+		cwd_fg = col_brmagenta,        cwd_bg = nil,
+		host_fg = col_magenta,         host_bg = nil,
+		timestamp_fg = col_magenta,
+	},
+
+	-- Monochrome Theme
+	["mono"] = {
+		cwd_fg = col_black,            cwd_bg = col_white,
+		host_fg = col_white,           host_bg = col_brblack,
+		timestamp_fg = col_white,
+	},
+
+	-- Monochrome Theme (No Background / No Fancy Fonts)
+	["mono_nb"] = {
+		cwd_fg = col_brwhite,          cwd_bg = nil,
+		host_fg = col_brblack,         host_bg = nil,
+		timestamp_fg = col_white,
+	},
+
+	-- Special Internal Themes/Colours - DO NOT USE / RENAME!
+	["muted"] =     { cwd_fg = col_black,  cwd_bg = col_brblack,  host_fg = col_brblack,  host_bg = col_black },
+	["muted_nb"] =  { cwd_fg = col_white,  cwd_bg = nil,          host_fg = col_brblack,  host_bg = nil },
+	["line2"] =     { fg = col_brwhite,    muted_fg = col_brblack },
+	["error"] =     { fg = col_brwhite,    bg = col_red },
+}
+
+--------------------------------------------------------------------------------
+-- Internal Data From here Onwards - Do Not Modify
+
+-- Default Settings (Change these through environment variables instead editing this file)
+local def_theme_name = "magenta_1"
+local def_enable_hostname = true
+local def_enable_timestamp = true
+local def_enable_errorind = true
+local def_enable_twoline = false
+local def_enable_greeting = true
+
+-- Constants, Special Codes and Internal Variables
+local sgr_esc = "\x1b["
+local sgr_bold = "1"
+local sgr_reverse = "0;7"
+local def_sep_hvbg = 0xe0b0
+local def_sep_nobg = 0x00bb
+local dw = { }
+
+--------------------------------------------------------------------------------
+-- Support Functions
+
+-- From agnoster.clinkprompt
+local function add_shift_and(x, y, s, a)
+    return x + bit32.band(bit32.rshift(y, s), a)
+end
+local function ucs2to8(c)
+    if type(c) ~= 'number' then error(string.format('arg #1 unexpected type %s; expected number', type(c)))
+    elseif c < 0 then error(string.format('arg #1 cannot be negative'))
+    elseif c <= 0x007f then return string.char(c)
+    elseif c <= 0x07ff then return string.char(add_shift_and(0xc0, c, 6, 0x1f), add_shift_and(0x80, c, 0, 0x3f))
+    elseif c <= 0xffff then return string.char(add_shift_and(0xe0, c, 12, 0x0f), add_shift_and(0x80, c, 6, 0x3f), add_shift_and(0x80, c, 0, 0x3f))
+    else error(string.format('arg #1 value 0x%x exceeds 0xffff', c))
+    end
+end
+
+-- Get the SGR Colour as a Foreground Colour
+-- Returns "Normal/Reset" if c = nil/false/empty
+local function sgr_fg(c)
+	c = c or ""
+	return sgr_esc..c.."m"
+end
+
+-- Get the SGR Colour as a Background Colour
+local function sgr_bg(c)
+	if (not c) then
+		return ""
+	end
+	c = (c + 10)
+	return sgr_esc..c.."m"
+end
+
+--------------------------------------------------------------------------------
+-- Query Data
+
+-- Get our current Path
+local function get_cwd()
+	return os.getcwd()
+	-- This is Windows, 'cd ~' doesn't work, so why bother...
```

Comment:
With Clink, `cd ~` should work.  Are you saying it isn't working right now?

Reply:
Ah no, I mean that by default Windows doesn't recognise `~` as home... there is a reason why clink is good stuff! But I wanted this prompt to be true to what Windows is.. shortcomings and all.. LOL...